### ISSUE TITLE
Automate AUR package release

### ISSUE THREAD

**Issue by badosu** [bryphe/oni#729]
When a new release is tagged, automate the process of updating the `PKGBUILD` version and also version it inside this repository.

Reference:

- https://github.com/tectonic-typesetting/tectonic/issues/54
- https://docs.travis-ci.com/user/deployment/releases/

I created this issue to track work and discussion on this matter, will work on this when possible.

**Comment by bryphe** [comment]
Thanks for tracking this, @badosu (and all your help on Arch) ! The approach tectonic is using looks pretty legit. Let me know if you need anything from me, or if you need any of the travis build settings tweaked.

**Comment by badosu** [comment]
@bryphe I noticed that the release tag for 0.2.10 has an `-recovery` appended. If we are going to automate this we'll have to use consistent names for the release tags. Noticed because I had to change the `PKGBUILD` manually for this.

**Comment by bryphe** [comment]
Sorry about that @badosu , I keep making your life difficult with the AUR repo... I'll switch to using a consistent release tag, and avoid using a `-recovery` tag in the future (I'll just bump the version).

I'll release a 0.2.11 build shortly with proper naming convention, that we can stick with moving forward.

**Comment by badosu** [comment]
Don't worry, this is not only for AUR but for any automation done by other actors.

**Comment by bryphe** [comment]
Right, it actually made it a pain updating the README.md with download links too 😞

**Comment by bryphe** [comment]
FYI, I just published [v0.2.11](https://github.com/bryphe/oni/releases/tag/v0.2.11) which has a consistent naming schema.

**Comment by badosu** [comment]
Thanks @bryphe, I was able to make the upgrade package to v0.2.10 anyway, next release I'll use the new schema

**Comment by bryphe** [comment]
Sounds great, thanks @badosu !

**Comment by badosu** [comment]
@bryphe First step should be to include the code for packaging it, then automating it later, where do you think would be a good place to put it?

I thought of `dist/arch` or something like that any ideas?

**Comment by bryphe** [comment]
Sweet! If there are scripts or code to run, maybe those could live in `build/aur`? Alternatively, there's a `scripts` folder. Would either of those work?

Today the stuff in the `dist` folder is solely built files (output from the various packaging strategies). I do a lot of `rimraf dist` so it might throw off my workflow if we have checked-in files there 😄

**Comment by someguynamedmatt** [comment]
@badosu I'm on Arch and I'd be willing to help out with this task, but I've never deployed to AUR before. You linked Tectonic and I was checking out [their deploy script](https://github.com/tectonic-typesetting/tectonic/blob/master/dist/arch/deploy.sh). And obviously we'd need to update the `travis.yml`. What else?

**Comment by CrossR** [comment]
From #2089, it looks like the AUR package hasn't been updated for the latest release, is it worth waiting for the PR, up manually updating for now?

**Comment by badosu** [comment]
Oh, I updated on the same day locally but forgot to push, fixed now. Thanks!
